src/models/Orders.js

```diff
@@ -0,0 +1,105 @@
+const mongoose = require('mongoose');
+
+// mongoose schema for the individual order items
+const orderItemSchema = new mongoose.Schema({
+    // The album field references the Album model and is required
+  album: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Album', // Referencing the Album model
+    required: true,
+  },
+  // The quantity field specifies the number of items in the order
+  quantity: {
+    type: Number,
+    required: true, // Quantity is required
+    min: 1, // Minimum quantity allowed is 1
+  },
+});
+
+// Create the OrderItem model using the defined schema
+const OrderItem = mongoose.model('OrderItem', orderItemSchema);
+
+// Schema for the main Order model
+const orderSchema = new mongoose.Schema({
+    // The user field references the User model and is required
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User', // Referencing the User model
+    required: true,
```

Comment:
For all the `required` fields, it's not necessary but can be helpful for error messaging, to include a message to describe what was required (eg: `required: [true, 'Please provide user']`)

Reply:
<img width="412" alt="screen" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/a7346c7b-116c-42e3-aaf5-026c2297aa75">

Reply:
We could add this type of messaging for each of the required fields

Reply:
<img width="671" alt="screen5" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/f91cda11-6c26-45a1-a505-f4fc6859a195">